scripts/scil_fodf_max_in_ventricles.py

```diff
@@ -40,25 +40,23 @@ def _build_arg_parser():
     p.add_argument('in_md',  metavar='MD',
                    help='Path to the mean diffusivity (MD) volume.')
 
-    p.add_argument('--fa_t', dest='fa_threshold',
-                   type=float, default='0.1',
-                   help='Maximal threshold of FA (voxels under that threshold'
-                        ' are considered for evaluation, [%(default)s]).')
-    p.add_argument('--md_t', dest='md_threshold',
-                   type=float, default='0.003',
+    p.add_argument('--fa_threshold', type=float, default='0.1',
```

Comment:
I would argue that `--fa_thr` is enough and shorter to write.

Reply:
Same for MD

Reply:
I can still write it -fa_t when you call your script, it will work. But I think it's clearer with the whole name. No?

Reply:
Alright, I was just to be consistent with other scripts, but at least I prefer --fa_threshold over --fa_t.